### DISCUSSION TITLE
Enabled SharedArrayBuffer for pages

### DISCUSSION THREAD

**Discussion by CoenraadS** [community/community#46419] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello

It would be nice if pages were default served with the following headers:

`"Cross-Origin-Embedder-Policy": "require-corp"`
`"Cross-Origin-Opener-Policy": "same-origin"`

So that Shared Array Buffers would work

**Comment by bluenote10** [comment]
See also https://github.com/community/community/discussions/13309
